xml/System.Security.Cryptography.X509Certificates/X509ChainElementCollection.xml

```diff
@@ -445,8 +445,8 @@
         <ReturnType>System.Object</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets an object that can be used to synchronize access to the collection.</summary>
+        <value>An object that can be used to synchronize access to the collection.</value>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Security.Cryptography.X509Certificates.X509ChainElementCollection> instance is cast to an <xref:System.Collections.ICollection> interface.  

           ]]></format>
         </remarks>
```